app/models/customer.rb

```diff
@@ -0,0 +1,8 @@
+class Customer < ApplicationRecord
+  has_many :customer_items
+  has_many :items, through: :customer_items
+
+  def add_item(item_id)
```

Comment:
Nice choice encapsulating this in a model method! Since it's just simple CRUD (.find), it's okay to have this in the controller but nothing wrong with having it here either.